README.md

```diff
@@ -98,17 +95,18 @@ See tracking issue: [curve25519-dalek/issues/521](https://github.com/dalek-crypt
 
 Curve arithmetic is implemented and used by selecting one of the following backends:
 
-| Backend            | Implementation                                                | Target backends             |
-| :---               | :---                                                          | :---                        |
-| `[default]`        | Automatic runtime backend selection (either serial or SIMD)   | `u32` <br/> `u64` <br/> `avx2` <br/> `avx512` |
-| `fiat`             | Formally verified field arithmetic from [fiat-crypto]         | `fiat_u32` <br/> `fiat_u64` |
+| Backend            | Selection | Implementation                                                | Target backends             |
+| :---               | :---      | :---                                                          | :---                        |
+| `serial`           | Automatic | serial software                                               | `u32` <br/> `u64`           |
+| `fiat`             | Manual    | Formally verified field arithmetic from [fiat-crypto]         | `fiat_u32` <br/> `fiat_u64` |
+| `simd`             | Automatic | Intel AVX2 / AVX512 IFMA accelerated backend                  | `avx2` <br/> `avx512`       |
```

Comment:
I might be a bit dense, but this confuses me as a reader. If I don't set _anything_ myself, what gets selected? IE what's the default? Also, if I set `curve25519_dalek_backend="simd"` on a non-simd system what happens? Or if I set `serial` on a `simd`-enabled system, will it "upgrade" to SIMD anyways because it's automatic?

I just reread the `build.rs` so I now know the answers to these questions, but they were honest ones I had from just the README.

Reply:
Also, `[default]` is still referenced below. And does the below "Word size for serial backends" need to be updated entirely?

Reply:
Ok I've clarified a bit - [Rendered](https://github.com/pinkforest/curve25519-dalek/blob/cleanup-features/README.md)